wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/internal/Util.kt

```diff
@@ -82,3 +82,9 @@ internal fun Int.isValidTag() =
 internal expect fun Char.isDigit(): Boolean
 
 internal expect fun String.toEnglishLowerCase(): String
+
+expect interface MutableQueue<T> : MutableCollection<T> {
+  fun poll(): T
```

Comment:
`T?`

Reply:
Probably, yes. And the `T` should be `T : Any`. Although I'm not sure if the `typealias` will match then. I'll try!